python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -0,0 +1,27 @@
+__all__ = ["SNCalculator", "KronFluxDivPsfFlux"]
+
+from lsst.pipe.tasks.dataFrameActions import DivideColumns
+
+
+class SNCalculator(DivideColumns):
+    """Calculate the signal to noise by default the i band PSF flux is used"""
```

Comment:
It isn't a config field because it is part of the column name in the object tables so I thought people would change the whole column name if they wanted a different band. I could make it a config option but then if people wanted a different type of flux they would have to make two changes.

Reply:
It's a minor point, but to me it seems like it would make the most sense to make both the `i` and the `PsFlux` parts configurable.  I guess it depends what the most likely configuration changes might be; if just choosing a different band of `PsFlux` would be significantly more common than changing the type of flux, then seems like it would make sense to have both `band` and `fluxType` to be config options.

Reply:
I'm not sure yet what usage will be most common, I would prefer to leave this for now until we know more about how people are going to use them. There will definitely be more tickets to change the actions.